src/rag/reranker/prompt.py

```diff
@@ -0,0 +1,31 @@
+RERANKER_PROMPT_TEMPLATE = """
+You are an expert in Kubernetes and Kyma.
+You are provided with a list of queries and a list of Kyma documents.
+Given the queries and documents, your task is to rank the documents based on their relevance to the queries.
+
+<input-documents>
+{documents}
+</input-documents>
+
+<input-queries>
+{queries}
+</input-queries>
+
+<your-tasks>
+1. Carefully read the queries and documents provided in the input sections.
+2. Compare the queries to each document by considering keywords or semantic meaning.
+3. Determine a relevance score for each document with respect to the queries (higher score means more relevant).
+4. Avoid documents that are irrelevant to the queries with very low relevance scores.
```

Comment:
maybe you can try this instruction:
`4. Filter out documents that have low or no relevance to the queries based on their relevance scores.`
I am not sure if it improves, but you can give a try. If nothing changes, you can keep yours.